source/install/install-rhel-66-mattermost.rst

```diff
@@ -0,0 +1,85 @@
+..  _install-rhel-66-mattermost:
+
+Installing Mattermost Server
+============================
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2
+
+**To install Mattermost Server on RHEL 6.6**
+
+1. Log into the server that will host Mattermost Server and open a terminal window.
+
+2. Download `the latest version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_. In the following command, replace ``X.X.X`` with the version that you want to download:
```

Comment:
Where can I find the comment? Sorry, I couldn't find it in the conversation here for some reason..